packages/flutter_test/test/animation_sheet_test.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('AnimationSheetBuilder', () {
+    testWidgets('correctly records frames', (WidgetTester tester) async {
+      final AnimationSheetBuilder builder = AnimationSheetBuilder(size: _DecuplePixels.size);
+
+      await tester.pumpFrames(
+        builder.record(
+          const _DecuplePixels(Duration(seconds: 1)),
+        ),
+        const Duration(milliseconds: 200),
+        const Duration(milliseconds: 100),
+      );
+
+      await tester.pumpFrames(
+        builder.record(
+          const _DecuplePixels(Duration(seconds: 1)),
+          recording: false,
+        ),
+        const Duration(milliseconds: 200),
+        const Duration(milliseconds: 100),
+      );
+
+      await tester.pumpFrames(
+        builder.record(
+          const _DecuplePixels(Duration(seconds: 1)),
+          recording: true,
+        ),
+        const Duration(milliseconds: 400),
+        const Duration(milliseconds: 100),
+      );
+
+      final Widget display = await builder.display();
+      await tester.binding.setSurfaceSize(builder.sheetSize());
+      await tester.pumpWidget(display);
+
+      await expectLater(find.byWidget(display), matchesGoldenFile('test.animation_sheet_builder.records.png'));
+    });
+
+    testWidgets('correctly wraps a row', (WidgetTester tester) async {
+      final AnimationSheetBuilder builder = AnimationSheetBuilder(size: const Size(12, 3));
+
+      const Duration duration = Duration(seconds: 2);
+      await tester.pumpFrames(
+        builder.record(const _DecuplePixels(duration)),
+        duration,
+        const Duration(milliseconds: 200),
+      );
+
+      final Widget display = await builder.display();
+      await tester.binding.setSurfaceSize(builder.sheetSize(maxWidth: 80));
+      await tester.pumpWidget(display);
+
+      await expectLater(find.byWidget(display), matchesGoldenFile('test.animation_sheet_builder.wraps.png'));
+    });
+  }, skip: isBrowser); // Animation sheet does not support browser https://github.com/flutter/flutter/issues/56001
```

Comment:
I think skips on groups don't work. Putting a skip on each test should resolve some of the failures here. Here's a reference: https://github.com/dart-lang/test/issues/545